internal/version/version.go

```diff
@@ -1,23 +1,70 @@
 package version
 
 import (
+	"bufio"
+	"io/ioutil"
 	"os"
 	"path/filepath"
 	"strings"
 
 	"github.com/blang/semver"
 	"github.com/google/renameio"
+	"github.com/pkg/errors"
 )
 
 // Version is the version of the build.
 const Version = "1.15.1-dev"
 
+// ShouldCrioWipe opens the version file, and parses it and the version string
+// If there is a parsing error, then crio should wipe, and the error is returned.
+// if parsing is successful, it compares the major and minor versions
+// and returns whether the major and minor versions are the same.
+// If they differ, then crio should wipe.
+func ShouldCrioWipe(versionFileName string) (bool, error) {
+	return shouldCrioWipe(versionFileName, Version)
+}
+
+// shouldCrioWipe is an internal function for testing purposes
+func shouldCrioWipe(versionFileName, versionString string) (bool, error) {
```

Comment:
I would change the return type to `bool` and only `logrus.Errorf` the errors.

Reply:
I'll do that for the public version, but I'd like to retain the errors for testing on the internal version

Reply:
on second though, I'd like to leave it as is. At this point, logrus isn't configured, and it's not included in version. I'd prefer to allow the caller to choose what to do with an error.

Reply:
SGTM